src/Microsoft.AspNet.Razor.Runtime/TagHelpers/ITagHelper.cs

```diff
@@ -11,6 +11,11 @@ namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
     public interface ITagHelper
     {
         /// <summary>
+        /// The value used to decided <see cref="ITagHelper"/> execution order. Lower values are executed first.
```

Comment:
- decided -> decide (no 'd')
- because the current text is a sentence fragment, better to change wording to `Gets the execution order of this <see cref="ITagHelper"/> relative to others targeting the same element.`
- expand second sentence -- `<see cref="ITagHelper"/> instances with lower values are executed first.`